pkg/segment/search/searchaggs.go

```diff
@@ -264,13 +272,22 @@ func addRecordToAggregations(grpReq *structs.GroupByRequest, timeHistogram *stru
 				}
 			}
 		} else {
+
+			// resize the working buf if we cannot accomodate the max value of any
+			// column's record
+			if len(aggsKeyWorkingBuf) < len(groupbyColKeyIndices)*65_000 {
+				aggsKeyWorkingBuf = toputils.ResizeSlice(aggsKeyWorkingBuf,
+					len(groupbyColKeyIndices)*65_000)
```

Comment:
nit: It would be better if we use variable or a constant such as `MAX_COL_VALUE_SIZE` or something instead of directly using `65_000`.

Reply:
yup fixed it in latest commit